package.json

```diff
@@ -132,9 +132,9 @@
     "react-lazyload": "3.2.1",
     "react-router-config": "5.1.1",
     "react-router-dom": "5.3.4",
-    "undici": "7.13.0",
+    "undici": "7.15.0",
     "url-parse": "1.5.10",
-    "winston": "patch:winston@3.8.2#./patches/winston-file-descriptor.patch"
+    "winston": "3.17.0"
```

Comment:
This will need to stay as `"winston": "patch:winston@3.8.2#./patches/winston-file-descriptor.patch"` as it uses a custom patch that we have applied over `3.8.2`